frontend/packages/data-portal/public/locales/en/translation.json

```diff
@@ -186,13 +191,15 @@
   "manual": "Manual",
   "meetsAll": "Meets all",
   "metadata": "Metadata",
+  "methodLinks": "Method Links",
   "methodType": "Method Type",
   "methodTypeAutomated": "<semibold>Automated Methods:</semibold> Any prediction that was not manually altered after being generated.",
   "methodTypeHybrid": "<semibold>Hybrid Methods:</semibold> Any annotation that has been curated/altered after generated with an automated method.",
```

Comment:
[Update tooltip copy to reflect final copy from dannielle:](https://docs.google.com/spreadsheets/d/1QEPiDcjGJYWSZXsbRjwHd9aeZjbk_tvv2UQ4DKyMwxk/edit?gid=339544814#gid=339544814) 
`Hybrid: Annotations were generated using a combination of automated and manual methods`